oshdb-filter/src/main/java/org/heigit/ohsome/oshdb/filter/Filter.java

```diff
@@ -12,7 +11,7 @@
 /**
  * A filter condition which can be applied to an OSM entity.
  */
-public interface Filter extends FilterExpression {
+public abstract class Filter {
```

Comment:
why the change from interface to abstract class?

Reply:
The change here is actually not to implement `FilterExpression` and to keep the Filter as a "sandalone" utility class, as it only provides static methods for creating `FilterExpressions` in various forms.